x-pack/solutions/security/plugins/security_solution/public/detection_engine/rule_creation_ui/components/step_define_rule/schema.tsx

```diff
@@ -68,6 +68,13 @@ export const schema: FormSchema<DefineStepRule> = {
         defaultMessage: 'Data view',
       }
     ),
+    helpText: i18n.translate(
+      'xpack.securitySolution.detectionEngine.createRule.stepAboutRule.dataViewSelector.helpText',
+      {
+        defaultMessage:
+          'Data views are displayed by name. To see their full definitions, visit Stack Management > Data Views.',
```

Comment:
Is there any way to link to the Data Views page in ESS and Serverless? The paths to the page are slightly different:
- **ESS**: Management > Stack Management > Kibana > Data Views 
- **Serverless**: Project Settings > Management > Data > Data Views

Reply:
@nastasha-solomon given that we link to the Data Views documentation at the top of the component:

<kbd><img width="1052" alt="Screenshot 2025-03-18 at 2 30 59 PM" src="https://github.com/user-attachments/assets/f630afb0-4086-46f4-bf49-b0f4b29a5a7a" /></kbd>

Perhaps we should just state the fact? In Discover, for example, "display by name" is the default with no explanation, and the fact of this issue/PR sort of implies that this is the de facto representation moving forward. Should we simply remove this help text entirely? We have opportunity to note the change in release notes, for any users that might be confused.

Reply:
I think it makes sense to move closer towards a unified UX - sourcerer is also moving towards utilizing the same component as Discover. 

I think the link to data view or index patterns also links to the wrong page. I think I remember there being a ticket about that.

Reply:
Yeah, that's a good point. If Discover changes the way it displays data view options (for example, it reverts back to displaying the definition instead of the name), we'd likely need to change the text again. I'm good with removing the help text entirely.

Reply:
@yctercero looks like it currently links to [these](https://www.elastic.co/guide/en/kibana/8.18/data-views.html) Kibana docs. Would you prefer that it link to [this](https://www.elastic.co/guide/en/security/current/data-views-in-sec.html) Security page insted?

Reply:
I removed the help text in https://github.com/elastic/kibana/pull/214495/commits/912e75aa901652c904b495785e125eac8a70dd3d. Please approve/deny based on your agreement.

Reply:
Looks good, thanks @rylnd !